galaxy_importer/schema.py

```diff
@@ -305,22 +305,71 @@ def _check_license_or_license_file(self):
             f"'license_file' ({self.license_file}) were invalid.")
 
 
+@attr.s(frozen=True)
+class CollectionArtifactFile(object):
+    name = attr.ib()
+    ftype = attr.ib()
+    src_name = attr.ib(default=None)
+    chksum_type = attr.ib(default="sha256")
+    chksum_sha256 = attr.ib(default=None)
+    format = attr.ib(default=1)
+
+
 @attr.s(frozen=True)
 class CollectionArtifactManifest(object):
     """Represents collection manifest metadata."""
 
     collection_info = attr.ib(type=CollectionInfo)
     format = attr.ib(default=1)
-    file_manifest_file = attr.ib(factory=dict)
+
+    file_manifest_file = attr.ib(default=None,
+                                 type=CollectionArtifactFile,
+                                 validator=attr.validators.optional(
+                                     attr.validators.instance_of(CollectionArtifactFile)))
 
     @classmethod
     def parse(cls, data):
         meta = json.loads(data)
         col_info = meta.pop('collection_info', None)
         meta['collection_info'] = CollectionInfo(**col_info)
+
+        # FIXME: Decide where we want missing info for FILES.json to raise an exception
```

Comment:
@awcrosby Can I assume a MANIFEST.json without a 'file_manifest_file' item should raise an exception? 

Updated to raise a `ValueError() from e` in parse, and the _load_collection_manifest() raises a
ManifestValidationError()  with chained exceptions to the ValueError and the KeyError

Added a unit test